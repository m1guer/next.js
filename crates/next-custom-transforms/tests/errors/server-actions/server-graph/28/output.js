export async function foo() {
    return 'data';
}
export async function bar() {
    return 'data';
}
export async function baz() {
    return 'data';
}
export async function qux() {
    return 'data';
}
export async function quux() {
    return 'data';
}
