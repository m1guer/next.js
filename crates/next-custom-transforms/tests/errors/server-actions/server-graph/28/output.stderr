  x Did you mean "use cache: remote"? "use cache:remote" is not a supported directive name."
  | 
   ,-[input.js:1:1]
 1 | 'use cache:remote'
   : ^^^^^^^^^^^^^^^^^^
   `----
  x Did you mean "use cache: default"? "use cache : default" is not a supported directive name."
  | 
   ,-[input.js:8:1]
 7 | export async function bar() {
 8 |   'use cache : default'
   :   ^^^^^^^^^^^^^^^^^^^^^
 9 |   return 'data'
   `----
  x Did you mean "use cache: private"? "use cache private" is not a supported directive name."
  | 
    ,-[input.js:13:1]
 12 | export async function baz() {
 13 |   'use cache private'
    :   ^^^^^^^^^^^^^^^^^^^
 14 |   return 'data'
    `----
  x Did you mean "use cache"? "use cache " is not a supported directive name."
  | 
    ,-[input.js:18:1]
 17 | export async function qux() {
 18 |   'use cache '
    :   ^^^^^^^^^^^^
 19 |   return 'data'
    `----
  x Did you mean "use cache: <cache-kind>"? "use cache: " is not a supported directive name."
  | 
    ,-[input.js:23:1]
 22 | export async function quux() {
 23 |   'use cache: '
    :   ^^^^^^^^^^^^^
 24 |   return 'data'
    `----
